README.md

```diff
@@ -43,7 +43,7 @@ You could get `tape` in three ways:
 
 #### Binary
 
-Execute `./tape -c config.yaml -n 40000` to generate 40000 transactions to Fabric.
+Execute `./tape run -c config.yaml -n 40000` to generate 40000 transactions to Fabric.
```

Comment:
why run?

Reply:
I think it's the default command as seen [there](https://github.com/Hyperledger-TWGC/tape/blob/master/cmd/tape/main.go#L21) besides if you print help the usage indicates the same